### ISSUE TITLE
documentation: Empty/missing doc on fx-hint

### ISSUE THREAD

**Issue by rkottmann** [Jinntec/Fore#274]
page https://jinntec.github.io/fore-docs/elements/ui/hint/ is just empty :(

**Comment by JoernT** [comment]
good point - actually we haven't used the element ourselves up-to-now and i rather think of removing it as it doesn't add much value over a HTML span with a class. It's a reminicense of the XForms hint element as was bulk-created back in the days. Opinions or ideas why fx-hint may still be worth it?

**Comment by JoernT** [comment]
reviewed again and IMO doesn't make sense any more - i marked fx-hint as deprecated.

**Comment by coloneltravis** [comment]
Could fx-hint be used to specify tooltip text to be displayed when a user hovers over a control?

**Comment by DrRataplan** [comment]
I do not think tooltips are a great pattern for most cases: they do not work nicely with screenreaders. From a Fore standpoint they are a mostly visual component and if you need them, (most) CSS frameworks have some way to display them. 

I found https://inclusive-components.design/tooltips-toggletips/ a good read. 

fx-hint does not feel like a good Fore component

**Comment by JoernT** [comment]
i close this ticket as the element is deprecated and better alternatives are around
